### PR TITLE
[PR] Migrate/QR component

### PR DESCRIPTION
**Pull request by denistsoi**
Resolves https://github.com/thenewboston-developers/ui-js/issues/12

(Doesn't include storybook for QR component, will do that in another PR).
acc: `baa336bca22c7590256a86c002e4dbe6c1f1c1f6c0a0fd3f0681eecb2cad0787`

Was a bit proactive in bumping the version here - (should just let @angle943 + co handle that next time 🙏 )

### CODE REVIEW

**Review by denistsoi — COMMENTED** [review]
package-lock was updated (to reflect update install and bumping patch version)

**Comment by denistsoi** [comment]
@angle943  - I've discovered that `tsdx test` somehow retains the state `toHaveBeenCalledTimes` - 

I'm going to move the `toHaveBeenCalledTimes` to the top of the testing order and will submit an issue in `tsdx`

Weird issue - but not something that should be resolve in this PR

**Comment by denistsoi** [comment]
I've submitted an issue to the previously failed test -  
https://github.com/formium/tsdx/issues/1007

**Comment by denistsoi** [comment]
Testing Evidence for QR storybook
Screenshot 2021-04-13 at 12 57 30 AM

**Comment by thesanjeevsharma** [comment]
Can you resolve conflicts on this one?

**Comment by thesanjeevsharma** [comment]
@denistsoi same changes as mentioned on PR #14

**Comment by denistsoi** [comment]
Yeah no problems @thesanjeevsharma - i'll do it tonight/tomorrow. 👍

File: src/components/Qr/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+const Qr: FC<QrProps> = ({ className, margin = 0, text, width = 140 }) => {
+  const [qr, setQr] = useState<ReactNode | null>(null);
+
+  useEffect(() => {
+    const generateQR = async (): Promise<void> => {
+      const url = await QrCode.toDataURL(text, {
+        color: {
+          dark: '#000000',
+          light: '#0000',
```
Comment (jamessspanggg) [inline src/components/Qr/index.tsx:27]: hmm the light color is an invalid color code? should it be `light: #ffffff` instead?

Comment (jamessspanggg) [reply src/components/Qr/index.tsx:27]: will need to change the tests if this was a mistake

Comment (denistsoi) [reply src/components/Qr/index.tsx:27]: Hmm - seems it's OG Bucky, i'll change that in the next commit. - (good catch btw @jamessspanggg)
https://github.com/thenewboston-developers/Website/blob/development/src/components/Qr/index.tsx#L20

**Review by jamessspanggg — APPROVED** [review]

**Comment by denistsoi** [comment]
@jamessspanggg - do we know when this will get merged - PR's been left open for a while

**Comment by jamessspanggg** [comment]
@angle943 can we merge this? all looks good from me

**Pull request merged**